3_Risk.md

```diff
@@ -2,71 +2,24 @@
 
 Activity Metric | Description
 --- | ---
-Contributor Importance | Percentage of commits by individual contributors
-Qualified Committers | Contributions over time and what components they commit to
-User Dependency | Number of users who are aware that they depend on the software
-Paid developers | Number of paid developers in community
+Contributor Importance | Percentage of commits by individual contributors over time

```

Comment:
Should it be Percentage of commits by individual contributors from certain organizations?  I think this sentence need clarity as all contributions are from individuals....

Reply:
Yea, that should be clarified in the definition. Thanks.